crates/tests/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "reth-tests"
```

Comment:
Most of these feel like the should be in a reth library which is then imported by the CLI?